src/mocks/create-permission-status.ts

```diff
@@ -11,6 +11,7 @@
  * @returns PermissionStatus.
  */
 export const createPermissionStatus = (state: PermissionState): PermissionStatus => ({
+  name: 'mock permission status',
```

Comment:
I had to add this to have tests passing locally, not sure why this was missing on master

Reply:
but then it fails in jenkins with it, so reverted it, something is weird here, but don't have time to investigate it now